docs/api/LightroomPartnerAPIsSpec.json

```diff
@@ -1283,6 +1283,11 @@
                   },
                   "/rels/rendition_type/2048": {
                     "href": "assets/<cover_asset_id>/renditions/2048"
+                  },
+                  "/rels/xmp/develop": {
+                    "href": "assets/<asset_id>/xmp/develop",
+                    "invalid": "true",
```

Comment:
@slohia-adobe @tarkik-adobe we should add documentation for meaning of `invalid`. invalid can serve as a note that a new edit is expected, the client could use the older one or poll after sometime for the new one